content/ux-guide/about.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About this guide
+permalink: /ux-guide/
+layout: layouts/page
+tags: ux-guide
+eleventyNavigation: 
+    parent: ux-guide
```

Comment:
For consistency, could you change these nested elements to use 2 space indents, to be consistent with the other guides?